plugins/module.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+module.exports = function (opts) {
+	const modules = opts && opts.modules !== null ? opts.modules : true;
+
+	if (typeof modules === 'string' && modules !== 'commonjs') {
+		throw new Error('@ava/stage-4 only support commonjs modules');
+	}
+
+	return modules ? require('./commonjs.json') : [];
```

Comment:
Feel free to return `['babel-plugin-transform-es2015-modules-commonjs']` directly. The JSON files are there so the plugin lists are easier to manage per Node.js version, but that doesn't apply here.